### PR TITLE
Added documentation for configuring promises

### PR DESCRIPTION
**Pull request by dr-impossible**

### CODE REVIEW

**Comment by bchociej** [comment]
I'm not sure this is strictly true:

> You'll need to explicitly enable promises on `aws-sdk-mock` if you use `promise()` functionality of the AWS SDK anywhere in your code.

Both `aws-sdk` and `aws-sdk-mock` work with Promises for me by automatically picking up the global Promise in Node, no configuration needed. Node 7 FWIW.

**Comment by dr-impossible** [comment]
Ok maybe it's not the case for modern versions of Node. I'm using the ancient 0.10 for a project, so I had to set this manually

**Comment by bchociej** [comment]
Ah that makes sense. I definitely think it's worth mentioning in the docs that you have to enable Promise support if you environment doesn't provide a global, or if you want to use a different implementation.

**Comment by dr-impossible** [comment]
Agreed. I've updated the PR

File: README.md [hunk 1]
```diff
@@ -167,6 +167,22 @@ AWS.setSDK(path.resolve('../../functions/foo/node_modules/aws-sdk'));
...
 ```
 
+### Configuring promises
+
+If your environment lacks a global Promise contstructor (e.g. nodejs 0.10), you can explicitly set the promises on `aws-sdk-mock`. Set the value of `AWS.Promise` to the constructor for your chosen promise library.
+
+Example (if Q is your promise library of choice):
+```js
+var AWS = require('aws-sdk-mock'),
+    Q = require('q');
```
Comment (nelsonic) [inline README.md]: @dr-impossible we _prefer_ one var statement per variable. see: https://github.com/dwyl/goodparts/issues/79

**Pull request merged**